pom.xml

```diff
@@ -1793,6 +1811,26 @@
                                 <compilerArgs combine.self="append">
                                     <compilerArg>-parameters</compilerArg>
                                 </compilerArgs>
+                                <testExcludes>
+                                    <testExclude>**/*NoParametersCompilerFlag*</testExclude>
```

Comment:
Is there any way you could use the directory name instead, e.g. `src/test/java17-noparameters/**`? So that we don't need tests to be named a certain way...